bedrock/firefox/templates/firefox/new/conformity-not-default/scene1.html

```diff
@@ -0,0 +1,48 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/fx-lifestyle-base.html" %}
+
+{% block page_css %}
+  {{ super() }}
+  {% stylesheet 'firefox_new_conformity_not_default' %}
+{% endblock %}
+
+{% block head_content %}
+  <h2>
+    You do you.
+  </h2>
+
+  <p>
+    We get what it’s like to be different. We’re the only non-corporate, non-conformist browser. We celebrate everybody’s right to break the mold, buck the trends, and demand better than the default.
+  </p>
+{% endblock %}
+
+{% block content_1 %}
+  <h3>Tune it up</h3>
+
+  <p>
+    Ad blockers, video downloaders, ghosters, grease monkeys, rocket launchers...Firefox has literally thousands of tools to dial in your web game just the way you like it.
+  </p>
+{% endblock %}
+
+{% block content_2 %}
+  <h3>Lock it up</h3>
+
+  <p>
+    Because we’re backed by a non-profit, we make your privacy the default. When you browse on Firefox, what you browse is your business.
+  </p>
+{% endblock %}
+
+{% block content_3 %}
+  <h3>Put a leopard on it</h3>
+
+  <p>
+    You may not believe this, but your browsing bar could have a leopard on it. Right now. Or Snoopy. Or Adriana Lima. Or it could just be aggressively purple. There’s a theme out there that’s so you.
```

Comment:
@FrancescoSTL per legal's comment we're can you replace with:

You may not believe this, but your browsing bar could have a leopard on it. Right now. Or a planet. Or a unicorn. Or it could just be aggressively purple. There’s a theme out there that’s so you.